### DISCUSSION TITLE
Hello, is there any context function in the latest version?

### DISCUSSION THREAD

**Discussion by nowindnowave** [acheong08/ChatGPT#1230] [General]
Is there a contextual use case? I remember earlier versions had option context

**Comment by acheong08** [comment]
Conversation history stored in memory for V3. Stored by conversation ID.

**Reply by nowindnowave** [reply]
Can I save it locally? I execute the following code every time
from revChatGPT.V3 import Chatbot
chatbot = Chatbot(api_key="<api_key>")
chatbot.ask("Hello world")

**Reply by acheong08** [reply]
https://github.com/acheong08/ChatGPT/wiki/V3#save

**Reply by nowindnowave** [reply]
Thank you

**Reply by nowindnowave** [reply]
God, I still don't understand how I call Chatbot.ask("hello") in my code to make him remember my session ID. I test that he doesn't remember the last time when I send a question, even if I pass the convo_id, gpt still doesn't understand,
If I refer to from revChatGPT.V3 import Chatbot through python, how can I call the previous session?

**Reply by acheong08** [reply]
Save and then load from file
